### ISSUE TITLE
issue running codecombat in docker on debian

### ISSUE THREAD

**Issue by Svetlana-T** [codecombat/codecombat#6345]
```
svetlana@thinkpenguin:~$ cd dev
svetlana@thinkpenguin:~/dev$ cd codecombat/
svetlana@thinkpenguin:~/dev/codecombat$ docker-co
docker-compose          docker-containerd-ctr   
docker-containerd       docker-containerd-shim  
svetlana@thinkpenguin:~/dev/codecombat$ docker-compose up
ERROR: Couldn't connect to Docker daemon at http+docker://localhost - is it running?

If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
svetlana@thinkpenguin:~/dev/codecombat$ nano docker-compose.yml 
svetlana@thinkpenguin:~/dev/codecombat$ dockerd
-bash: dockerd: команда не найдена
svetlana@thinkpenguin:~/dev/codecombat$ file /usr/sbin/dockerd
/usr/sbin/dockerd: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, Go BuildID=l7l_2-AJiaVMlxe4ebrD/wg3WWnG19oMIzVYORZoP/siPVMHrZeAk2Hwitcu6t/ODibfeb7zn9PvIFSCqDV, BuildID[sha1]=16b2bc5541fce90c342efc2df352756f2b04ed7a, stripped
svetlana@thinkpenguin:~/dev/codecombat$ sudo dockerd
[sudo] пароль для svetlana: 
Error starting daemon: pid file found, ensure docker is not running or delete /var/run/docker.pid
svetlana@thinkpenguin:~/dev/codecombat$ ps aux | grep docker
root      1882  0.2  0.5 1296024 81896 ?       Ssl  08:03   0:29 /usr/sbin/dockerd -H fd://
root      1903  0.4  0.2 1321248 33560 ?       Ssl  08:03   0:43 docker-containerd --config /var/run/docker/containerd/containerd.toml --log-level info
svetlana 15627  0.0  0.0   6224   876 pts/8    S+   11:03   0:00 grep docker
svetlana@thinkpenguin:~/dev/codecombat$ sudo docker-compose up
Creating network "codecombat_default" with the default driver
Building proxy
Step 1/9 : FROM node:8.15.1-jessie
8.15.1-jessie: Pulling from library/node
bf295113f40d: Pull complete
62fe5b9a5ae4: Pull complete
6cc848917b0a: Pull complete
053381643ee3: Pull complete
46b0950c9f62: Pull complete
fc436e75e314: Pull complete
951b875a8ad2: Pull complete
Digest: sha256:09cdf0008759bb5f76de95f0851834fbc5670215563c5c88b1ae526ed519725c
Status: Downloaded newer image for node:8.15.1-jessie
 ---> 27ea6de4c599
Step 2/9 : ARG RUBY_VERSION=2.6.1
 ---> Running in 4abaa224317e
Removing intermediate container 4abaa224317e
 ---> 1b4dfb11bb36
Step 3/9 : ENV NPM_GLOBAL_PREFIX=/npm
 ---> Running in f997afcef429
Removing intermediate container f997afcef429
 ---> 2791d2dea167
Step 4/9 : WORKDIR /coco
 ---> Running in a3e4d61c2ca8
Removing intermediate container a3e4d61c2ca8
 ---> e9dbb5b299b7
Step 5/9 : ENV PATH $PATH:/root/.rbenv/bin:/root/.rbenv/shims
 ---> Running in f39c15a30854
Removing intermediate container f39c15a30854
 ---> 05dff3b8ab6f
Step 6/9 : RUN apt-get update &&     apt-get install -y git curl libssl-dev libreadline-dev zlib1g-dev autoconf bison                        build-essential libyaml-dev libncurses5-dev                        libffi-dev libgdbm-dev &&     curl -sL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-installer | bash -
 ---> Running in 223247842d6c
Ign http://deb.debian.org jessie InRelease
Get:1 http://deb.debian.org jessie Release.gpg [1652 B]
Get:2 http://security.debian.org jessie/updates InRelease [44.9 kB]
Get:3 http://deb.debian.org jessie Release [77.3 kB]
Get:4 http://security.debian.org jessie/updates/main amd64 Packages [992 kB]
Get:5 http://deb.debian.org jessie/main amd64 Packages [9098 kB]
Fetched 10.2 MB in 17s (593 kB/s)
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
autoconf is already the newest version.
libgdbm-dev is already the newest version.
libffi-dev is already the newest version.
libyaml-dev is already the newest version.
libncurses5-dev is already the newest version.
libreadline-dev is already the newest version.
The following extra packages will be installed:
  libbison-dev libcurl3 libcurl4-openssl-dev libssl1.0.0 zlib1g
Suggested packages:
  bison-doc gettext-base git-daemon-run git-daemon-sysvinit git-doc git-el
  git-email git-gui gitk gitweb git-arch git-cvs git-mediawiki git-svn
  libcurl4-doc libcurl3-dbg libidn11-dev libldap2-dev librtmp-dev
  libssh2-1-dev
Recommended packages:
  less rsync libssl-doc
The following NEW packages will be installed:
  bison build-essential libbison-dev
The following packages will be upgraded:
  curl git libcurl3 libcurl4-openssl-dev libssl-dev libssl1.0.0 zlib1g
  zlib1g-dev
8 upgraded, 3 newly installed, 0 to remove and 92 not upgraded.
Need to get 7852 kB of archives.
After this operation, 2766 kB of additional disk space will be used.
Get:1 http://deb.debian.org/debian/ jessie/main libbison-dev amd64 2:3.0.2.dfsg-2 [426 kB]
Get:2 http://security.debian.org/debian-security/ jessie/updates/main zlib1g-dev amd64 1:1.2.8.dfsg-2+deb8u1 [206 kB]
Get:3 http://deb.debian.org/debian/ jessie/main bison amd64 2:3.0.2.dfsg-2 [765 kB]
Get:4 http://deb.debian.org/debian/ jessie/main build-essential amd64 11.7 [7114 B]
Get:5 http://security.debian.org/debian-security/ jessie/updates/main zlib1g amd64 1:1.2.8.dfsg-2+deb8u1 [88.3 kB]
Get:6 http://security.debian.org/debian-security/ jessie/updates/main libssl-dev amd64 1.0.1t-1+deb8u12 [1284 kB]
Get:7 http://security.debian.org/debian-security/ jessie/updates/main libssl1.0.0 amd64 1.0.1t-1+deb8u12 [1049 kB]
Get:8 http://security.debian.org/debian-security/ jessie/updates/main libcurl4-openssl-dev amd64 7.38.0-4+deb8u16 [338 kB]
Get:9 http://security.debian.org/debian-security/ jessie/updates/main curl amd64 7.38.0-4+deb8u16 [202 kB]
Get:10 http://security.debian.org/debian-security/ jessie/updates/main libcurl3 amd64 7.38.0-4+deb8u16 [261 kB]
Get:11 http://security.debian.org/debian-security/ jessie/updates/main git amd64 1:2.1.4-2.1+deb8u10 [3227 kB]
debconf: delaying package configuration, since apt-utils is not installed
Fetched 7852 kB in 3s (2462 kB/s)
(Reading database ... 21694 files and directories currently installed.)
Preparing to unpack .../zlib1g-dev_1%3a1.2.8.dfsg-2+deb8u1_amd64.deb ...
Unpacking zlib1g-dev:amd64 (1:1.2.8.dfsg-2+deb8u1) over (1:1.2.8.dfsg-2+b1) ...
Preparing to unpack .../zlib1g_1%3a1.2.8.dfsg-2+deb8u1_amd64.deb ...
Unpacking zlib1g:amd64 (1:1.2.8.dfsg-2+deb8u1) over (1:1.2.8.dfsg-2+b1) ...
Setting up zlib1g:amd64 (1:1.2.8.dfsg-2+deb8u1) ...
Processing triggers for libc-bin (2.19-18+deb8u10) ...
(Reading database ... 21692 files and directories currently installed.)
Preparing to unpack .../libssl-dev_1.0.1t-1+deb8u12_amd64.deb ...
Unpacking libssl-dev:amd64 (1.0.1t-1+deb8u12) over (1.0.1t-1+deb8u11) ...
Preparing to unpack .../libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb ...
Unpacking libssl1.0.0:amd64 (1.0.1t-1+deb8u12) over (1.0.1t-1+deb8u11) ...
Preparing to unpack .../libcurl4-openssl-dev_7.38.0-4+deb8u16_amd64.deb ...
Unpacking libcurl4-openssl-dev:amd64 (7.38.0-4+deb8u16) over (7.38.0-4+deb8u14) ...
Preparing to unpack .../curl_7.38.0-4+deb8u16_amd64.deb ...
Unpacking curl (7.38.0-4+deb8u16) over (7.38.0-4+deb8u14) ...
Preparing to unpack .../libcurl3_7.38.0-4+deb8u16_amd64.deb ...
Unpacking libcurl3:amd64 (7.38.0-4+deb8u16) over (7.38.0-4+deb8u14) ...
Selecting previously unselected package libbison-dev:amd64.
Preparing to unpack .../libbison-dev_2%3a3.0.2.dfsg-2_amd64.deb ...
Unpacking libbison-dev:amd64 (2:3.0.2.dfsg-2) ...
Selecting previously unselected package bison.
Preparing to unpack .../bison_2%3a3.0.2.dfsg-2_amd64.deb ...
Unpacking bison (2:3.0.2.dfsg-2) ...
Selecting previously unselected package build-essential.
Preparing to unpack .../build-essential_11.7_amd64.deb ...
Unpacking build-essential (11.7) ...
Preparing to unpack .../git_1%3a2.1.4-2.1+deb8u10_amd64.deb ...
Unpacking git (1:2.1.4-2.1+deb8u10) over (1:2.1.4-2.1+deb8u7) ...
Setting up zlib1g-dev:amd64 (1:1.2.8.dfsg-2+deb8u1) ...
Setting up libssl1.0.0:amd64 (1.0.1t-1+deb8u12) ...
debconf: unable to initialize frontend: Dialog
debconf: (TERM is not set, so the dialog frontend is not usable.)
debconf: falling back to frontend: Readline
Setting up libssl-dev:amd64 (1.0.1t-1+deb8u12) ...
Setting up libcurl3:amd64 (7.38.0-4+deb8u16) ...
Setting up libcurl4-openssl-dev:amd64 (7.38.0-4+deb8u16) ...
Setting up curl (7.38.0-4+deb8u16) ...
Setting up libbison-dev:amd64 (2:3.0.2.dfsg-2) ...
Setting up bison (2:3.0.2.dfsg-2) ...
update-alternatives: using /usr/bin/bison.yacc to provide /usr/bin/yacc (yacc) in auto mode
Setting up build-essential (11.7) ...
Setting up git (1:2.1.4-2.1+deb8u10) ...
Processing triggers for libc-bin (2.19-18+deb8u10) ...
Installing rbenv with git...
Initialized empty Git repository in /root/.rbenv/.git/
Updating origin
From https://github.com/rbenv/rbenv
 * [new branch]      master     -> origin/master
 * [new tag]         v0.1.0     -> v0.1.0
 * [new tag]         v0.1.1     -> v0.1.1
 * [new tag]         v0.1.2     -> v0.1.2
 * [new tag]         v0.2.0     -> v0.2.0
 * [new tag]         v0.2.1     -> v0.2.1
 * [new tag]         v0.3.0     -> v0.3.0
 * [new tag]         v0.4.0     -> v0.4.0
 * [new tag]         v1.0.0     -> v1.0.0
 * [new tag]         v1.1.0     -> v1.1.0
 * [new tag]         v1.1.1     -> v1.1.1
 * [new tag]         v1.1.2     -> v1.1.2
Already on 'master'
Branch master set up to track remote branch master from origin.
make: Entering directory '/root/.rbenv/src'
gcc -fPIC     -c -o realpath.o realpath.c
gcc -shared -Wl,-soname,../libexec/rbenv-realpath.dylib  -o ../libexec/rbenv-realpath.dylib realpath.o 
make: Leaving directory '/root/.rbenv/src'

Installing ruby-build with git...
Cloning into '/root/.rbenv/plugins/ruby-build'...

All done!
Note that this installer does NOT edit your shell configuration files:
1. You'll want to ensure that `~/.rbenv/bin' is added to PATH.
2. Run `rbenv init' to view instructions on how to configure rbenv for your shell.
3. Launch a new terminal window after editing shell configuration files.
4. (Optional) Run the doctor command to verify the installation:
   wget -q "https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-doctor" -O- | bash

Removing intermediate container 223247842d6c
 ---> ffc3fe543f36
Step 7/9 : RUN rbenv install ${RUBY_VERSION} &&     rbenv global ${RUBY_VERSION}
 ---> Running in 0cc58cff5cc9
Downloading ruby-2.6.1.tar.bz2...
-> https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.1.tar.bz2
Installing ruby-2.6.1...
Installed ruby-2.6.1 to /root/.rbenv/versions/2.6.1

Removing intermediate container 0cc58cff5cc9
 ---> 1eac68a2c5c9
Step 8/9 : RUN mkdir $NPM_GLOBAL_PREFIX &&     npm config set prefix $NPM_GLOBAL_PREFIX --global
 ---> Running in 126498e10976
Removing intermediate container 126498e10976
 ---> 3b482efdec3b
Step 9/9 : COPY . .
 ---> ff3a5fbd4ef3
Successfully built ff3a5fbd4ef3
Successfully tagged codecombat_proxy:latest
WARNING: Image for service proxy was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
Creating codecombat_proxy_1 ... done
Attaching to codecombat_proxy_1
proxy_1  | 
proxy_1  | > uws@9.14.0 install /coco/node_modules/uws
proxy_1  | > node-gyp rebuild > build_log.txt 2>&1 || exit 0
proxy_1  | 
proxy_1  | 
proxy_1  | > node-sass@4.13.1 install /coco/node_modules/node-sass
proxy_1  | > node scripts/install.js
proxy_1  | 
proxy_1  | Downloading binary from https://github.com/sass/node-sass/releases/download/v4.13.1/linux-x64-57_binding.node
proxy_1  | Download complete
proxy_1  | Binary saved to /coco/node_modules/node-sass/vendor/linux-x64-57/binding.node
proxy_1  | Caching binary to /root/.npm/node-sass/4.13.1/linux-x64-57_binding.node
proxy_1  | 
proxy_1  | > core-js@2.6.12 postinstall /coco/node_modules/@babel/polyfill/node_modules/core-js
proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
proxy_1  | 
proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
proxy_1  | 
proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
proxy_1  | > https://opencollective.com/core-js 
proxy_1  | > https://www.patreon.com/zloirock 
proxy_1  | 
proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
proxy_1  | 
proxy_1  | 
proxy_1  | > core-js@2.6.11 postinstall /coco/node_modules/babel-polyfill/node_modules/core-js
proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
proxy_1  | 
proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
proxy_1  | 
proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
proxy_1  | > https://opencollective.com/core-js 
proxy_1  | > https://www.patreon.com/zloirock 
proxy_1  | 
proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
proxy_1  | 
proxy_1  | 
proxy_1  | > core-js@2.6.11 postinstall /coco/node_modules/babel-runtime/node_modules/core-js
proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
proxy_1  | 
proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
proxy_1  | 
proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
proxy_1  | > https://opencollective.com/core-js 
proxy_1  | > https://www.patreon.com/zloirock 
proxy_1  | 
proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
proxy_1  | 
proxy_1  | 
proxy_1  | > core-js@2.6.11 postinstall /coco/node_modules/karma/node_modules/core-js
proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
proxy_1  | 
proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
proxy_1  | 
proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
proxy_1  | > https://opencollective.com/core-js 
proxy_1  | > https://www.patreon.com/zloirock 
proxy_1  | 
proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
proxy_1  | 
proxy_1  | 
proxy_1  | > core-js@3.6.4 postinstall /coco/node_modules/core-js
proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
proxy_1  | 
proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
proxy_1  | 
proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
proxy_1  | > https://opencollective.com/core-js 
proxy_1  | > https://www.patreon.com/zloirock 
proxy_1  | 
proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
proxy_1  | 
proxy_1  | 
proxy_1  | > node-sass@4.13.1 postinstall /coco/node_modules/node-sass
proxy_1  | > node scripts/build.js
proxy_1  | 
proxy_1  | Binary found at /coco/node_modules/node-sass/vendor/linux-x64-57/binding.node
proxy_1  | Testing binary
proxy_1  | Binary is fine
proxy_1  | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.1 (node_modules/@babel/cli/node_modules/fsevents):
proxy_1  | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
proxy_1  | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/watchpack/node_modules/fsevents):
proxy_1  | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
proxy_1  | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
proxy_1  | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
proxy_1  | 
proxy_1  | added 1955 packages from 1431 contributors and audited 2025 packages in 56.106s
proxy_1  | found 1279 vulnerabilities (633 low, 46 moderate, 584 high, 16 critical)
proxy_1  |   run `npm audit fix` to fix them, or `npm audit` for details
proxy_1  | 
proxy_1  | > codecombat@1.0.0 proxy /coco
proxy_1  | > COCO_PROXY='true' nodemon
proxy_1  | 
proxy_1  | 10 Jul 01:11:39 - [nodemon] v1.6.1
proxy_1  | 10 Jul 01:11:39 - [nodemon] reading config /coco/nodemon.json
proxy_1  | 10 Jul 01:11:39 - [nodemon] to restart at any time, enter `rs`
proxy_1  | 10 Jul 01:11:39 - [nodemon] ignoring: /coco/.git/**/* /coco/node_modules/**/* bower_components .sass-cache
proxy_1  | 10 Jul 01:11:39 - [nodemon] watching: server_config.js server_setup.coffee /coco/app/schemas/**/* ./server app/assets/main.html
proxy_1  | 10 Jul 01:11:39 - [nodemon] watching extensions: js,coffee
proxy_1  | 10 Jul 01:11:39 - [nodemon] starting `node index.js`
proxy_1  | 10 Jul 01:11:39 - [nodemon] child pid: 610
proxy_1  | 10 Jul 01:11:39 - [nodemon] watching 177 files
proxy_1  | Using dev proxy server
proxy_1  | fs.js:948
proxy_1  |   binding.stat(pathModule._makeLong(path));
proxy_1  |           ^
proxy_1  | 
proxy_1  | Error: ENOENT: no such file or directory, stat '/coco/public/images/favicon/favicon.ico'
proxy_1  |     at Object.fs.statSync (fs.js:948:11)
proxy_1  |     at resolveSync (/coco/node_modules/serve-favicon/index.js:187:17)
proxy_1  |     at favicon (/coco/node_modules/serve-favicon/index.js:61:12)
proxy_1  |     at setupExpressMiddleware (/coco/server_setup.coffee:86:11)
proxy_1  |     at Object.exports.setupMiddleware (/coco/server_setup.coffee:201:3)
proxy_1  |     at module.exports.createAndConfigureApp (/coco/server.coffee:26:15)
proxy_1  |     at Object.module.exports.startServer (/coco/server.coffee:14:9)
proxy_1  |     at Object.<anonymous> (/coco/index.js:12:8)
proxy_1  |     at Module._compile (module.js:653:30)
proxy_1  |     at Object.Module._extensions..js (module.js:664:10)
proxy_1  |     at Module.load (module.js:566:32)
proxy_1  |     at tryModuleLoad (module.js:506:12)
proxy_1  |     at Function.Module._load (module.js:498:3)
proxy_1  |     at Function.Module.runMain (module.js:694:10)
proxy_1  |     at startup (bootstrap_node.js:204:16)
proxy_1  |     at bootstrap_node.js:625:3
proxy_1  | 
proxy_1  | 10 Jul 01:11:40 - [nodemon] app crashed - waiting for file changes before starting...
```

**Comment by ashliakhov** [comment]
Hi, devs, first of all, thanks for the possibility to run CodeCombat locally.

But why don't you fix the Docker image for such a long time?
As far as I understand images and containers are supposed to run on any kind of environment and return exactly the same result. That's the whole idea behind it.

You got lot's of questions and issues regarding failing containers for quite a long time, but nothing is fixed.

Could you please take a look?

Thank you in advance!

**Comment by Svetlana-T** [comment]
Hi, have you tried to use the docker image on debian, @ashliakhov ? It is everyone's, if you reproduce the problem, maybe it would 1) help others see it is not an issue on my pc and 2) we would have two our heads trying to fix it rather than one mine. Thanks

**Comment by RoaringBeluga** [comment]
I confirm the issue on Manjaro. Definitely a lot of things fail while building so it looks like assets are not generated properly and http://localhost:7777 shows precisely nothing. If, as suggested in the discussion of another issue on this tracker, I copy the *favicon.ico* by hand, the failure is still there.
I attached below the log of ***docker-compose up --build*** command. My hunch is, some (most?) dependencies are not installed, leading to the whole thing's spectacular failure. 

[coco-build_up.log](https://github.com/codecombat/codecombat/files/6804041/coco-build_up.log)

**Comment by souhaibdjaballah** [comment]
> ```
> svetlana@thinkpenguin:~$ cd dev
> svetlana@thinkpenguin:~/dev$ cd codecombat/
> svetlana@thinkpenguin:~/dev/codecombat$ docker-co
> docker-compose          docker-containerd-ctr   
> docker-containerd       docker-containerd-shim  
> svetlana@thinkpenguin:~/dev/codecombat$ docker-compose up
> ERROR: Couldn't connect to Docker daemon at http+docker://localhost - is it running?
> 
> If it's at a non-standard location, specify the URL with the DOCKER_HOST environment variable.
> svetlana@thinkpenguin:~/dev/codecombat$ nano docker-compose.yml 
> svetlana@thinkpenguin:~/dev/codecombat$ dockerd
> -bash: dockerd: команда не найдена
> svetlana@thinkpenguin:~/dev/codecombat$ file /usr/sbin/dockerd
> /usr/sbin/dockerd: ELF 64-bit LSB shared object, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, Go BuildID=l7l_2-AJiaVMlxe4ebrD/wg3WWnG19oMIzVYORZoP/siPVMHrZeAk2Hwitcu6t/ODibfeb7zn9PvIFSCqDV, BuildID[sha1]=16b2bc5541fce90c342efc2df352756f2b04ed7a, stripped
> svetlana@thinkpenguin:~/dev/codecombat$ sudo dockerd
> [sudo] пароль для svetlana: 
> Error starting daemon: pid file found, ensure docker is not running or delete /var/run/docker.pid
> svetlana@thinkpenguin:~/dev/codecombat$ ps aux | grep docker
> root      1882  0.2  0.5 1296024 81896 ?       Ssl  08:03   0:29 /usr/sbin/dockerd -H fd://
> root      1903  0.4  0.2 1321248 33560 ?       Ssl  08:03   0:43 docker-containerd --config /var/run/docker/containerd/containerd.toml --log-level info
> svetlana 15627  0.0  0.0   6224   876 pts/8    S+   11:03   0:00 grep docker
> svetlana@thinkpenguin:~/dev/codecombat$ sudo docker-compose up
> Creating network "codecombat_default" with the default driver
> Building proxy
> Step 1/9 : FROM node:8.15.1-jessie
> 8.15.1-jessie: Pulling from library/node
> bf295113f40d: Pull complete
> 62fe5b9a5ae4: Pull complete
> 6cc848917b0a: Pull complete
> 053381643ee3: Pull complete
> 46b0950c9f62: Pull complete
> fc436e75e314: Pull complete
> 951b875a8ad2: Pull complete
> Digest: sha256:09cdf0008759bb5f76de95f0851834fbc5670215563c5c88b1ae526ed519725c
> Status: Downloaded newer image for node:8.15.1-jessie
>  ---> 27ea6de4c599
> Step 2/9 : ARG RUBY_VERSION=2.6.1
>  ---> Running in 4abaa224317e
> Removing intermediate container 4abaa224317e
>  ---> 1b4dfb11bb36
> Step 3/9 : ENV NPM_GLOBAL_PREFIX=/npm
>  ---> Running in f997afcef429
> Removing intermediate container f997afcef429
>  ---> 2791d2dea167
> Step 4/9 : WORKDIR /coco
>  ---> Running in a3e4d61c2ca8
> Removing intermediate container a3e4d61c2ca8
>  ---> e9dbb5b299b7
> Step 5/9 : ENV PATH $PATH:/root/.rbenv/bin:/root/.rbenv/shims
>  ---> Running in f39c15a30854
> Removing intermediate container f39c15a30854
>  ---> 05dff3b8ab6f
> Step 6/9 : RUN apt-get update &&     apt-get install -y git curl libssl-dev libreadline-dev zlib1g-dev autoconf bison                        build-essential libyaml-dev libncurses5-dev                        libffi-dev libgdbm-dev &&     curl -sL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-installer | bash -
>  ---> Running in 223247842d6c
> Ign http://deb.debian.org jessie InRelease
> Get:1 http://deb.debian.org jessie Release.gpg [1652 B]
> Get:2 http://security.debian.org jessie/updates InRelease [44.9 kB]
> Get:3 http://deb.debian.org jessie Release [77.3 kB]
> Get:4 http://security.debian.org jessie/updates/main amd64 Packages [992 kB]
> Get:5 http://deb.debian.org jessie/main amd64 Packages [9098 kB]
> Fetched 10.2 MB in 17s (593 kB/s)
> Reading package lists...
> Reading package lists...
> Building dependency tree...
> Reading state information...
> autoconf is already the newest version.
> libgdbm-dev is already the newest version.
> libffi-dev is already the newest version.
> libyaml-dev is already the newest version.
> libncurses5-dev is already the newest version.
> libreadline-dev is already the newest version.
> The following extra packages will be installed:
>   libbison-dev libcurl3 libcurl4-openssl-dev libssl1.0.0 zlib1g
> Suggested packages:
>   bison-doc gettext-base git-daemon-run git-daemon-sysvinit git-doc git-el
>   git-email git-gui gitk gitweb git-arch git-cvs git-mediawiki git-svn
>   libcurl4-doc libcurl3-dbg libidn11-dev libldap2-dev librtmp-dev
>   libssh2-1-dev
> Recommended packages:
>   less rsync libssl-doc
> The following NEW packages will be installed:
>   bison build-essential libbison-dev
> The following packages will be upgraded:
>   curl git libcurl3 libcurl4-openssl-dev libssl-dev libssl1.0.0 zlib1g
>   zlib1g-dev
> 8 upgraded, 3 newly installed, 0 to remove and 92 not upgraded.
> Need to get 7852 kB of archives.
> After this operation, 2766 kB of additional disk space will be used.
> Get:1 http://deb.debian.org/debian/ jessie/main libbison-dev amd64 2:3.0.2.dfsg-2 [426 kB]
> Get:2 http://security.debian.org/debian-security/ jessie/updates/main zlib1g-dev amd64 1:1.2.8.dfsg-2+deb8u1 [206 kB]
> Get:3 http://deb.debian.org/debian/ jessie/main bison amd64 2:3.0.2.dfsg-2 [765 kB]
> Get:4 http://deb.debian.org/debian/ jessie/main build-essential amd64 11.7 [7114 B]
> Get:5 http://security.debian.org/debian-security/ jessie/updates/main zlib1g amd64 1:1.2.8.dfsg-2+deb8u1 [88.3 kB]
> Get:6 http://security.debian.org/debian-security/ jessie/updates/main libssl-dev amd64 1.0.1t-1+deb8u12 [1284 kB]
> Get:7 http://security.debian.org/debian-security/ jessie/updates/main libssl1.0.0 amd64 1.0.1t-1+deb8u12 [1049 kB]
> Get:8 http://security.debian.org/debian-security/ jessie/updates/main libcurl4-openssl-dev amd64 7.38.0-4+deb8u16 [338 kB]
> Get:9 http://security.debian.org/debian-security/ jessie/updates/main curl amd64 7.38.0-4+deb8u16 [202 kB]
> Get:10 http://security.debian.org/debian-security/ jessie/updates/main libcurl3 amd64 7.38.0-4+deb8u16 [261 kB]
> Get:11 http://security.debian.org/debian-security/ jessie/updates/main git amd64 1:2.1.4-2.1+deb8u10 [3227 kB]
> debconf: delaying package configuration, since apt-utils is not installed
> Fetched 7852 kB in 3s (2462 kB/s)
> (Reading database ... 21694 files and directories currently installed.)
> Preparing to unpack .../zlib1g-dev_1%3a1.2.8.dfsg-2+deb8u1_amd64.deb ...
> Unpacking zlib1g-dev:amd64 (1:1.2.8.dfsg-2+deb8u1) over (1:1.2.8.dfsg-2+b1) ...
> Preparing to unpack .../zlib1g_1%3a1.2.8.dfsg-2+deb8u1_amd64.deb ...
> Unpacking zlib1g:amd64 (1:1.2.8.dfsg-2+deb8u1) over (1:1.2.8.dfsg-2+b1) ...
> Setting up zlib1g:amd64 (1:1.2.8.dfsg-2+deb8u1) ...
> Processing triggers for libc-bin (2.19-18+deb8u10) ...
> (Reading database ... 21692 files and directories currently installed.)
> Preparing to unpack .../libssl-dev_1.0.1t-1+deb8u12_amd64.deb ...
> Unpacking libssl-dev:amd64 (1.0.1t-1+deb8u12) over (1.0.1t-1+deb8u11) ...
> Preparing to unpack .../libssl1.0.0_1.0.1t-1+deb8u12_amd64.deb ...
> Unpacking libssl1.0.0:amd64 (1.0.1t-1+deb8u12) over (1.0.1t-1+deb8u11) ...
> Preparing to unpack .../libcurl4-openssl-dev_7.38.0-4+deb8u16_amd64.deb ...
> Unpacking libcurl4-openssl-dev:amd64 (7.38.0-4+deb8u16) over (7.38.0-4+deb8u14) ...
> Preparing to unpack .../curl_7.38.0-4+deb8u16_amd64.deb ...
> Unpacking curl (7.38.0-4+deb8u16) over (7.38.0-4+deb8u14) ...
> Preparing to unpack .../libcurl3_7.38.0-4+deb8u16_amd64.deb ...
> Unpacking libcurl3:amd64 (7.38.0-4+deb8u16) over (7.38.0-4+deb8u14) ...
> Selecting previously unselected package libbison-dev:amd64.
> Preparing to unpack .../libbison-dev_2%3a3.0.2.dfsg-2_amd64.deb ...
> Unpacking libbison-dev:amd64 (2:3.0.2.dfsg-2) ...
> Selecting previously unselected package bison.
> Preparing to unpack .../bison_2%3a3.0.2.dfsg-2_amd64.deb ...
> Unpacking bison (2:3.0.2.dfsg-2) ...
> Selecting previously unselected package build-essential.
> Preparing to unpack .../build-essential_11.7_amd64.deb ...
> Unpacking build-essential (11.7) ...
> Preparing to unpack .../git_1%3a2.1.4-2.1+deb8u10_amd64.deb ...
> Unpacking git (1:2.1.4-2.1+deb8u10) over (1:2.1.4-2.1+deb8u7) ...
> Setting up zlib1g-dev:amd64 (1:1.2.8.dfsg-2+deb8u1) ...
> Setting up libssl1.0.0:amd64 (1.0.1t-1+deb8u12) ...
> debconf: unable to initialize frontend: Dialog
> debconf: (TERM is not set, so the dialog frontend is not usable.)
> debconf: falling back to frontend: Readline
> Setting up libssl-dev:amd64 (1.0.1t-1+deb8u12) ...
> Setting up libcurl3:amd64 (7.38.0-4+deb8u16) ...
> Setting up libcurl4-openssl-dev:amd64 (7.38.0-4+deb8u16) ...
> Setting up curl (7.38.0-4+deb8u16) ...
> Setting up libbison-dev:amd64 (2:3.0.2.dfsg-2) ...
> Setting up bison (2:3.0.2.dfsg-2) ...
> update-alternatives: using /usr/bin/bison.yacc to provide /usr/bin/yacc (yacc) in auto mode
> Setting up build-essential (11.7) ...
> Setting up git (1:2.1.4-2.1+deb8u10) ...
> Processing triggers for libc-bin (2.19-18+deb8u10) ...
> Installing rbenv with git...
> Initialized empty Git repository in /root/.rbenv/.git/
> Updating origin
> From https://github.com/rbenv/rbenv
>  * [new branch]      master     -> origin/master
>  * [new tag]         v0.1.0     -> v0.1.0
>  * [new tag]         v0.1.1     -> v0.1.1
>  * [new tag]         v0.1.2     -> v0.1.2
>  * [new tag]         v0.2.0     -> v0.2.0
>  * [new tag]         v0.2.1     -> v0.2.1
>  * [new tag]         v0.3.0     -> v0.3.0
>  * [new tag]         v0.4.0     -> v0.4.0
>  * [new tag]         v1.0.0     -> v1.0.0
>  * [new tag]         v1.1.0     -> v1.1.0
>  * [new tag]         v1.1.1     -> v1.1.1
>  * [new tag]         v1.1.2     -> v1.1.2
> Already on 'master'
> Branch master set up to track remote branch master from origin.
> make: Entering directory '/root/.rbenv/src'
> gcc -fPIC     -c -o realpath.o realpath.c
> gcc -shared -Wl,-soname,../libexec/rbenv-realpath.dylib  -o ../libexec/rbenv-realpath.dylib realpath.o 
> make: Leaving directory '/root/.rbenv/src'
> 
> Installing ruby-build with git...
> Cloning into '/root/.rbenv/plugins/ruby-build'...
> 
> All done!
> Note that this installer does NOT edit your shell configuration files:
> 1. You'll want to ensure that `~/.rbenv/bin' is added to PATH.
> 2. Run `rbenv init' to view instructions on how to configure rbenv for your shell.
> 3. Launch a new terminal window after editing shell configuration files.
> 4. (Optional) Run the doctor command to verify the installation:
>    wget -q "https://github.com/rbenv/rbenv-installer/raw/HEAD/bin/rbenv-doctor" -O- | bash
> 
> Removing intermediate container 223247842d6c
>  ---> ffc3fe543f36
> Step 7/9 : RUN rbenv install ${RUBY_VERSION} &&     rbenv global ${RUBY_VERSION}
>  ---> Running in 0cc58cff5cc9
> Downloading ruby-2.6.1.tar.bz2...
> -> https://cache.ruby-lang.org/pub/ruby/2.6/ruby-2.6.1.tar.bz2
> Installing ruby-2.6.1...
> Installed ruby-2.6.1 to /root/.rbenv/versions/2.6.1
> 
> Removing intermediate container 0cc58cff5cc9
>  ---> 1eac68a2c5c9
> Step 8/9 : RUN mkdir $NPM_GLOBAL_PREFIX &&     npm config set prefix $NPM_GLOBAL_PREFIX --global
>  ---> Running in 126498e10976
> Removing intermediate container 126498e10976
>  ---> 3b482efdec3b
> Step 9/9 : COPY . .
>  ---> ff3a5fbd4ef3
> Successfully built ff3a5fbd4ef3
> Successfully tagged codecombat_proxy:latest
> WARNING: Image for service proxy was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.
> Creating codecombat_proxy_1 ... done
> Attaching to codecombat_proxy_1
> proxy_1  | 
> proxy_1  | > uws@9.14.0 install /coco/node_modules/uws
> proxy_1  | > node-gyp rebuild > build_log.txt 2>&1 || exit 0
> proxy_1  | 
> proxy_1  | 
> proxy_1  | > node-sass@4.13.1 install /coco/node_modules/node-sass
> proxy_1  | > node scripts/install.js
> proxy_1  | 
> proxy_1  | Downloading binary from https://github.com/sass/node-sass/releases/download/v4.13.1/linux-x64-57_binding.node
> proxy_1  | Download complete
> proxy_1  | Binary saved to /coco/node_modules/node-sass/vendor/linux-x64-57/binding.node
> proxy_1  | Caching binary to /root/.npm/node-sass/4.13.1/linux-x64-57_binding.node
> proxy_1  | 
> proxy_1  | > core-js@2.6.12 postinstall /coco/node_modules/@babel/polyfill/node_modules/core-js
> proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
> proxy_1  | 
> proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
> proxy_1  | 
> proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> proxy_1  | > https://opencollective.com/core-js 
> proxy_1  | > https://www.patreon.com/zloirock 
> proxy_1  | 
> proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
> proxy_1  | 
> proxy_1  | 
> proxy_1  | > core-js@2.6.11 postinstall /coco/node_modules/babel-polyfill/node_modules/core-js
> proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
> proxy_1  | 
> proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
> proxy_1  | 
> proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> proxy_1  | > https://opencollective.com/core-js 
> proxy_1  | > https://www.patreon.com/zloirock 
> proxy_1  | 
> proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
> proxy_1  | 
> proxy_1  | 
> proxy_1  | > core-js@2.6.11 postinstall /coco/node_modules/babel-runtime/node_modules/core-js
> proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
> proxy_1  | 
> proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
> proxy_1  | 
> proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> proxy_1  | > https://opencollective.com/core-js 
> proxy_1  | > https://www.patreon.com/zloirock 
> proxy_1  | 
> proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
> proxy_1  | 
> proxy_1  | 
> proxy_1  | > core-js@2.6.11 postinstall /coco/node_modules/karma/node_modules/core-js
> proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
> proxy_1  | 
> proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
> proxy_1  | 
> proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> proxy_1  | > https://opencollective.com/core-js 
> proxy_1  | > https://www.patreon.com/zloirock 
> proxy_1  | 
> proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
> proxy_1  | 
> proxy_1  | 
> proxy_1  | > core-js@3.6.4 postinstall /coco/node_modules/core-js
> proxy_1  | > node -e "try{require('./postinstall')}catch(e){}"
> proxy_1  | 
> proxy_1  | Thank you for using core-js ( https://github.com/zloirock/core-js ) for polyfilling JavaScript standard library!
> proxy_1  | 
> proxy_1  | The project needs your help! Please consider supporting of core-js on Open Collective or Patreon: 
> proxy_1  | > https://opencollective.com/core-js 
> proxy_1  | > https://www.patreon.com/zloirock 
> proxy_1  | 
> proxy_1  | Also, the author of core-js ( https://github.com/zloirock ) is looking for a good job -)
> proxy_1  | 
> proxy_1  | 
> proxy_1  | > node-sass@4.13.1 postinstall /coco/node_modules/node-sass
> proxy_1  | > node scripts/build.js
> proxy_1  | 
> proxy_1  | Binary found at /coco/node_modules/node-sass/vendor/linux-x64-57/binding.node
> proxy_1  | Testing binary
> proxy_1  | Binary is fine
> proxy_1  | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.1 (node_modules/@babel/cli/node_modules/fsevents):
> proxy_1  | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.1: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
> proxy_1  | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/watchpack/node_modules/fsevents):
> proxy_1  | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
> proxy_1  | npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.7 (node_modules/fsevents):
> proxy_1  | npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.7: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
> proxy_1  | 
> proxy_1  | added 1955 packages from 1431 contributors and audited 2025 packages in 56.106s
> proxy_1  | found 1279 vulnerabilities (633 low, 46 moderate, 584 high, 16 critical)
> proxy_1  |   run `npm audit fix` to fix them, or `npm audit` for details
> proxy_1  | 
> proxy_1  | > codecombat@1.0.0 proxy /coco
> proxy_1  | > COCO_PROXY='true' nodemon
> proxy_1  | 
> proxy_1  | 10 Jul 01:11:39 - [nodemon] v1.6.1
> proxy_1  | 10 Jul 01:11:39 - [nodemon] reading config /coco/nodemon.json
> proxy_1  | 10 Jul 01:11:39 - [nodemon] to restart at any time, enter `rs`
> proxy_1  | 10 Jul 01:11:39 - [nodemon] ignoring: /coco/.git/**/* /coco/node_modules/**/* bower_components .sass-cache
> proxy_1  | 10 Jul 01:11:39 - [nodemon] watching: server_config.js server_setup.coffee /coco/app/schemas/**/* ./server app/assets/main.html
> proxy_1  | 10 Jul 01:11:39 - [nodemon] watching extensions: js,coffee
> proxy_1  | 10 Jul 01:11:39 - [nodemon] starting `node index.js`
> proxy_1  | 10 Jul 01:11:39 - [nodemon] child pid: 610
> proxy_1  | 10 Jul 01:11:39 - [nodemon] watching 177 files
> proxy_1  | Using dev proxy server
> proxy_1  | fs.js:948
> proxy_1  |   binding.stat(pathModule._makeLong(path));
> proxy_1  |           ^
> proxy_1  | 
> proxy_1  | Error: ENOENT: no such file or directory, stat '/coco/public/images/favicon/favicon.ico'
> proxy_1  |     at Object.fs.statSync (fs.js:948:11)
> proxy_1  |     at resolveSync (/coco/node_modules/serve-favicon/index.js:187:17)
> proxy_1  |     at favicon (/coco/node_modules/serve-favicon/index.js:61:12)
> proxy_1  |     at setupExpressMiddleware (/coco/server_setup.coffee:86:11)
> proxy_1  |     at Object.exports.setupMiddleware (/coco/server_setup.coffee:201:3)
> proxy_1  |     at module.exports.createAndConfigureApp (/coco/server.coffee:26:15)
> proxy_1  |     at Object.module.exports.startServer (/coco/server.coffee:14:9)
> proxy_1  |     at Object.<anonymous> (/coco/index.js:12:8)
> proxy_1  |     at Module._compile (module.js:653:30)
> proxy_1  |     at Object.Module._extensions..js (module.js:664:10)
> proxy_1  |     at Module.load (module.js:566:32)
> proxy_1  |     at tryModuleLoad (module.js:506:12)
> proxy_1  |     at Function.Module._load (module.js:498:3)
> proxy_1  |     at Function.Module.runMain (module.js:694:10)
> proxy_1  |     at startup (bootstrap_node.js:204:16)
> proxy_1  |     at bootstrap_node.js:625:3
> proxy_1  | 
> proxy_1  | 10 Jul 01:11:40 - [nodemon] app crashed - waiting for file changes before starting...
> ```

Same issue on Ubuntu 20.04.

**Comment by ZJUSTKIDDING** [comment]
Had this problem, tried to fix it, but I can't login to sign the CLA. Edit the start command in docker-compose.yml like this:
`command: bash -c "npm install && npm install bower && npm run proxy"` OR `command: bash -c "npm install && npm run build && npm run proxy"`

**Comment by patr1ce** [comment]
Hey !
Same problem here on a Debian 11 after few modifications :
- add "--force-yes" in the Dockerfile apt-get install to have it execute
- Update version of node and ruby since they were too old and the install crashes
Now, got to the same error with the following logs (same with both "npm install bower" or "npm run build" in the docker-compose.yml) :
- in the docker-compose logs :
```
proxy_1  | [nodemon] 2.0.19
proxy_1  | [nodemon] reading config ./nodemon.json
proxy_1  | [nodemon] to restart at any time, enter `rs`
proxy_1  | [nodemon] or send SIGHUP to 95 to restart
proxy_1  | [nodemon] watching path(s): server_config.js server_setup.coffee app/schemas/**/* server app/assets/main.html
proxy_1  | [nodemon] watching extensions: js,coffee
proxy_1  | [nodemon] starting `node index.js`
proxy_1  | [nodemon] forking
proxy_1  | [nodemon] child pid: 108
proxy_1  | [nodemon] watching 180 files
proxy_1  | Using dev proxy server
proxy_1  | Error. Couldn't find /coco/public_coco/images/favicon/favicon-coco. It is likely that the public_coco folder is not built. Try:
proxy_1  |
proxy_1  |   npm run build
proxy_1  |
proxy_1  | for an initial build, or
proxy_1  |
proxy_1  |   npm run dev
proxy_1  |
proxy_1  | for live rebuilding of your front-end changes. If those don't work, make sure you are running the correct version of node and have installed all dependencies with:
proxy_1  |
proxy_1  |   npm install --also=dev
proxy_1  |
proxy_1  | [nodemon] app crashed - waiting for file changes before starting...
```

- a "npm run build" finish without any problem

- a "npm run start" gives de following error

```
verbose stack Error: codecombat@1.0.0 start: `node ./index.js`
verbose stack Exit status 1
verbose stack     at EventEmitter.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycle/index.js:332:16)
verbose stack     at EventEmitter.emit (events.js:400:28)
verbose stack     at ChildProcess.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-lifecycleb/spawn.js:55:14)
verbose stack     at ChildProcess.emit (events.js:400:28)
verbose stack     at maybeClose (internal/child_process.js:1088:16)
verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:296:5)
verbose pkgid codecombat@1.0.0
verbose cwd /coco
verbose Linux 5.10.0-19-cloud-amd64
verbose argv "/usr/local/bin/node" "/usr/local/bin/npm" "run" "start"
verbose node v14.21.2
verbose npm  v6.14.17
error code ELIFECYCLE
error errno 1
error codecombat@1.0.0 start: `node ./index.js`
error Exit status 1
error Failed at the codecombat@1.0.0 start script.
error This is probably not a problem with npm. There is likely additional logging output above.
verbose exit [ 1, true ]
```
